Modified viewer loop to fix visualization when using multimodel

p-shg opened the pull request:
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document [docs/contribution.md]?
* [ ] Have you checked to ensure there aren't other open [Pull Requests] for the same update/change?
* [ ] Have you opened/linked the issue related to your pull request?
* [ ] Have you used the tag [WIP] for on-going changes, and removed it when the pull request was ready?
* [ ] When ready to merge, have you sent a comment pinging @pariterre in it?

### New Feature Submissions:

1. [ ] Does your submission pass the tests (if not please explain why this is intended)?
2. [ ] Did you write a proper documentation (docstrings and ReadMe)
3. [ ] Have you linted your code locally prior to submission (using the command: `black . -l120 --exclude "external/*"`)?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new examples for your core changes, as applicable?
* [ ] Have you written new tests for your core changes, as applicable?

The changes fix the visualization for the  torque_driven_ocp/example_multi_biorbd_model.py
Thanks to @EveCharbie for the guidance

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pyomeca/bioptim/1012)
<!-- Reviewable:end -->

pariterre reviewed (changes requested):
> @pariterre reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pyomeca/bioptim/1012)* status: all files reviewed, 2 unresolved discussions (waiting on @p-shg)

___
*[`bioptim/models/biorbd/viewer_pyorerun.py` line 90 at r1](https://reviewable.io/reviews/pyomeca/bioptim/1012#-O_Cwdb86bQirYQMEgJ2:-O_Cwdb86bQirYQMEgJ3:b-gc1vzl) ([raw file](https://github.com/pyomeca/bioptim/blob/514de37ea3e1fdc25cde6c443d2ea7e4b539a273/bioptim/models/biorbd/viewer_pyorerun.py#L90)):*
> ```Python
>     show_now: Bool = True,
>     tracked_markers: list[NpArray] = None,
>     models: BiorbdModel | list[BiorbdModel] = None,
> ```

BiorbdModel | list[BiorbdModel] | list[MultiBiorbdModel]
___
*[`bioptim/models/biorbd/viewer_pyorerun.py` line 112 at r1](https://reviewable.io/reviews/pyomeca/bioptim/1012#-O_CwgNT9lqLArUeYxDQ:-O_CwgNT9lqLArUeYxDR:bf5e9w4) ([raw file](https://github.com/pyomeca/bioptim/blob/514de37ea3e1fdc25cde6c443d2ea7e4b539a273/bioptim/models/biorbd/viewer_pyorerun.py#L112)):*
> ```Python
>         prerun.add_phase(t_span=data["time"], phase=idx_phase)
> 
>         if isinstance(model, biorbd.Model):
> ```

elif isinstance...
for m in model:
...


<!-- Sent from Reviewable.io -->

p-shg commented:
@pariterre I updated this code, it should be all good regarding your comments
However I added changes from a different PR by mistake, how do I fix this other than deleting the files and recommitting? (or is this the way?)
Thanks

pariterre commented:
Deleting the file is not the way as it remembers that this file should be deleted. Depending on how deep you are in your commit, sometimes it can be to reset and cherry-pick commits, or to accept the mistake haha

pariterre reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> @pariterre reviewed 11 of 11 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pyomeca/bioptim/1012#-:-Oa544ULAzNZ8tcWYJMx:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @p-shg)



<!-- Sent from Reviewable.io -->
